docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -32,34 +32,81 @@ environment for debugging. The tool is meant to be used for debugging a single
 Tetragon agent node. Note that in the context of Kubernetes, the command needs
 to be run from inside the Tetragon Pod's container.
 
+**Key Information Collected by the Bugtool:**
+
+- Tetragon configuration
+- Network configuration
+- Kernel configuration
+- eBPF maps
+- Process traces (if tracing is enabled)
+
+## Running tetra bugtool Demonstrations
+
+### Kubernetes
+
+1. **Identify Tetragon Pod:**
+
+```bash
+kubectl get pods -n <tetragron-namespace> -l app.kubernetes.io/name=tetragon
+```
+
+2. **Execute tetra bugtool within the Pod:**
+
+```bash
+kubectl exec -n <tetragron-namespace> <tetragron-pod-name> -- tetra bugtool
+```
+
+### Container Installation
+
+1. **Enter the Tetragon Container:**
+
+```bash
+docker exec -it <tetragron-container-id> /bin/sh
+```
+
+2. **Run tetra bugtool:**
+
+```bash
+tetra bugtool
+```
+
+### Systemd Host Installation
+
+1. **Execute tetra bugtool with Elevated Permissions:**
+
+```bash
+sudo tetra bugtool
+```
+
+> Note: Replace <...> placeholders with the actual values applicable to your setup.
```

Comment:
You can use the note shortcode:
```suggestion
{{< note >}}
Replace <...> placeholders with the actual values applicable to your setup.
{{< /note >}}
```

Reply:
I wonder if it is working as it isn't reflected in the preview.

Reply:
you should be able to see it here https://deploy-preview-2115--tetragon.netlify.app/docs/troubleshooting/ :)